aml-development/ozp-center
# 20180416 delete category validate modal

AMLOS-602

**Description:**
Displays information needed by the App Steward when determining the impact prior to deleting a category.  Allows the App Steward to identify potential orphans, and displays the current status of category listing associations

Copy and paste Acceptance Criteria from original issue
[  ] Displays listing and its associated category count 
[  ] Displays a link/button to the 'Intention to Delete Category' notification
[  ] Displays a link to the 'Bulk Category Update form.

**How to test code**
- Pull the branch into ozp-center
- Open AML  & log-in as bigbrother
- Navigate to the Menu-->Manage --> Center Settings : Category Tab
- Select a category and click the 'Validate" button located on the grid toolbar.
-    The Validate window (modal) is displayed
- To verify the data:   The displayed 'Listing' and 'number of category associations' can be obtained from the following Endpoint:
          **GET /api/category/7/listing/**    (replace '7' with the appropriate category_id)
           
     _**NOTE:**_ The endpoint was tested in a previous PR.   
     See- Backend: PR#409  20180206 bulk category            
     [https://github.com/aml-development/ozp-backend/pull/409  
          

Please Review
@aml-development/ozp-front-team 

Checklist

 Read CONTRIBUTING.md
 Write code to complete task
 All tests must pass before merging the pull request
 At least 2 people reviewed code
